share/doc/homebrew/Acceptable-Formulae.md

```diff
@@ -66,9 +66,8 @@ Formulae in the core repository must have a stable version tagged by
 the upstream project. Tarballs are preferred to git checkouts, and
 tarballs should include the version in the filename whenever possible.
 
-Software that does not provide a stable, tagged version, or had guidance to
-always install the most recent version, should be put in
-[homebrew/head-only](https://github.com/Homebrew/homebrew-head-only) or [homebrew/devel-only](https://github.com/Homebrew/homebrew-devel-only).
+Software that does not provide a stable, tagged version should be put in
```

Comment:
Sounds good. I'd be tempted to just make `development/beta` into `unstable`, but I don't think it makes much difference.

On the terrible UX I'd probably give reasons, partially so if we all get hit by a truck the next set of maintainers know what we thought about fixing, and partially because someone's bound to ask.

Just something like there being no automatic update mechanism for `head-only` formulae which makes them very quickly outdated, they regularly break due to upstream changes, we can't provide bottles for them, etc, maybe.